addressing.mss

```diff
@@ -32,11 +32,13 @@
   }
 }
 
+// ircama: revised zoom and text size
```

Comment:
@ircama: There's no need to leave comments like this in the code. "git blame" can be used to show when, why and who made the last change, if we ever need to know it.